examples/FactoryExample.php

```diff
@@ -11,13 +11,15 @@
 $anotherException = new Exception('baz', 400);
 $jsonErrorException = new JsonApiErrorException('a custom message', 0, null, 'foo', '504', ['bar' => 'baz'], 'testlink');
 
+$jsonApiErrorFactory = new JsonApiErrorFactory(true);
```

Comment:
Same with the tests - maybe add an example without debug-flag and one with it? 🤔